cuda_bindings/cuda/bindings/_path_finder/README.md

```diff
@@ -0,0 +1,61 @@
+# `cuda.bindings.path_finder` Module
+
+## Public API (Work in Progress)
+
+Currently exposes two primary interfaces:
+
+```
+cuda.bindings.path_finder._SUPPORTED_LIBNAMES  # ('nvJitLink', 'nvrtc', 'nvvm')
+cuda.bindings.path_finder._load_nvidia_dynamic_library(libname: str) -> LoadedDL
+```
+
+**Note:**
+These APIs are prefixed with an underscore because they are considered
+experimental while undergoing active development, although already
+reasonably well-tested through CI pipelines.
+
+## Library Loading Search Priority
+
+The `load_nvidia_dynamic_library()` function implements a hierarchical search
+strategy for locating NVIDIA shared libraries:
+
```

Comment:
I think we're missing using the `CUDA_HOME` / `CUDA_PATH` environment variables?

Reply:
I added commit 3277ac52e0f79cdd3a1b19d39f828aeb992a2457.

At this stage (but not in the long run), I'm leaving `CUDA_HOME` / `CUDA_PATH` to cuda_paths.py, which references both.

There is a bigger theme on my mind: I look at cuda_paths.py as a can of worms. It's a can that's in production and mostly works. Trying to open it in this PR could easily make us get lost in the weeds.